task/dice-data/src/main.rs

```diff
@@ -0,0 +1,85 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#![no_std]
+#![no_main]
+
+use dice::{AliasData, Cert, SeedBuf};
+use ringbuf::{ringbuf, ringbuf_entry};
+use salty::{
+    constants::{PUBLICKEY_SERIALIZED_LENGTH, SIGNATURE_SERIALIZED_LENGTH},
+    signature::{Keypair, Signature},
+    PublicKey,
+};
+use userlib::{hl, UnwrapLite};
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+enum Trace {
+    PubCheck(bool),
+    AliasSigCheck(bool),
+    DeviceIdSigCheck(bool),
+    Fwid([u8; 32]),
+    AllChecksPassed,
+    None,
+}
+
+ringbuf!(Trace, 64, Trace::None);
```

Comment:
This will take up a moderate amount of RAM (~2K).  Probably not a big deal, but worth looking at if we run low.